repo/packages/H/hue/0/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "object",
+  "properties": {
+    "hue/configuration": {
```

Comment:
We now recommend people to use hierarchical options files. See Marathon and Cassandra. Thanks!